docs/getting-started/setup.md

```diff
@@ -9,83 +9,88 @@ import TabItem from '@theme/TabItem';
 
-There are two ways of installing the node:
+      ### (1) Download the Binary
+      Go to the [ink-node releases page](https://github.com/use-ink/ink-node/releases). Under `Assets` of the latest release, download the appropriate binary for your platform:
+         - **Linux (ARM64)**: `ink-node-linux-arm64.tar.gz`
+         - **Linux (x86)**: `ink-node-linux.tar.gz`  
+         - **macOS**: `ink-node-mac-universal.tar.gz`
 
-### (1) Download the Binary
-Go to the [ink-node releases page](https://github.com/use-ink/ink-node/releases). Under `Assets` of the latest release, download the appropriate binary for your platform:
-   - **Linux (ARM64)**: `ink-node-linux-arm64.tar.gz`
-   - **Linux (x86)**: `ink-node-linux.tar.gz`  
-   - **macOS**: `ink-node-mac-universal.tar.gz`
+      Make the binary executable:
+         ```bash
+         chmod +x ./ink-node
+         ```
 
-Make the binary executable:
-   ```bash
-   chmod +x ./ink-node
-   ```
+      **For macOS users:**
+      When you first try to run `ink-node`, macOS may show a security warning.
 
-**For macOS users:**
-When you first try to run `ink-node`, macOS may show a security warning.
+      To allow the binary to run:
+      1. Click the **question mark (?)** icon at the right top corner of the warning.
+      2. Follow the Apple instructions that appear.
+      3. Try running `ink-node` again and click **"Open Anyway"** when prompted.
 
-To allow the binary to run:
-1. Click the **question mark (?)** icon at the right top corner of the warning.
-2. Follow the Apple instructions that appear.
-3. Try running `ink-node` again and click **"Open Anyway"** when prompted.
-
-To confirm that `ink-node` is working correctly, run:
-```bash
-./ink-node --version
-```
-If you see version information, your installation is successful!
+      To confirm that `ink-node` is working correctly, run:
+      ```bash
+      ./ink-node --version
+      ```
+      If you see version information, your installation is successful!
 
-### (2) Build it yourself
+      ### (2) Install or Build it yourself
 
-Alternatively, you can build the node by yourself.
-This can take a while though!
+      Alternatively, you can build the node by yourself.
+      This can take a while though!
 
-The build instructions and pre-requisites can be found
-[here](https://github.com/use-ink/ink-node?tab=readme-ov-file#build-locally).
\ No newline at end of file
+      The build instructions and pre-requisites can be found
+      [here](https://github.com/use-ink/ink-node?tab=readme-ov-file#build-locally).
+  </TabItem>
+    <TabItem value="pop" label="Pop">
+       Use the [Pop CLI](https://learn.onpop.io/contracts/welcome/install-pop-cli) for ink! smart contract development with an enhanced developer experience.
+
+       ```bash
+       cargo install --git https://github.com/r0gue-io/pop-cli.git --branch  v6.0.0-alpha.3 --no-default-features --locked -F polkavm-contracts,chain,telemetry
```

Comment:
I'm not sure about this one, but if you release in sync with ink! it should be `alpha.4` here.
```suggestion
       cargo install --git https://github.com/r0gue-io/pop-cli.git --branch v6.0.0-alpha.4 --no-default-features --locked -F polkavm-contracts,chain,telemetry
```